File: To-Do_List_Manager.cs
```CSharp
using System;
using System.Collections.Generic;

class Program
{
    static void Main(string[] args)
    {
        List<string> toDoList = new List<string>();
        string command = "";

        while (command != "exit")
        {
            Console.WriteLine("Enter command (add, remove, view, exit): ");
            command = Console.ReadLine();

            switch (command)
            {
                case "add":
                    Console.WriteLine("Enter task: ");
                    string task = Console.ReadLine();
                    toDoList.Add(task);
                    break;

                case "remove":
                    Console.WriteLine("Enter task number to remove: ");
                    int taskNumber = Convert.ToInt32(Console.ReadLine());
                    if (taskNumber > 0 && taskNumber <= toDoList.Count)
                    {
                        toDoList.RemoveAt(taskNumber - 1);
                    }
                    else
                    {
                        Console.WriteLine("Invalid task number.");
                    }
                    break;

                case "view":
                    for (int i = 0; i < toDoList.Count; i++)
                    {
                        Console.WriteLine((i + 1) + ". " + toDoList[i]);
                    }
                    break;

                case "exit":
                    break;

                default:
                    Console.WriteLine("Invalid command.");
                    break;
            }
        }
    }
}
```

File: To-Do_List_Manager.java
```java
import java.util.ArrayList;
import java.util.Scanner;

public class ToDoListManager {
    private ArrayList<String> toDoList = new ArrayList<>();

    public void addTask(String task) {
        toDoList.add(task);
    }

    public void removeTask(int taskIndex) {
        toDoList.remove(taskIndex);
    }

    public void displayTasks() {
        for (int i = 0; i < toDoList.size(); i++) {
            System.out.println((i + 1) + ". " + toDoList.get(i));
        }
    }

    public static void main(String[] args) {
        ToDoListManager manager = new ToDoListManager();
        Scanner scanner = new Scanner(System.in);
        String command;
        do {
            System.out.println("Enter command (add, remove, display, exit): ");
            command = scanner.nextLine();
            switch (command) {
                case "add":
                    System.out.println("Enter task: ");
                    String task = scanner.nextLine();
                    manager.addTask(task);
                    break;
                case "remove":
                    System.out.println("Enter task number: ");
                    int taskIndex = scanner.nextInt();
                    scanner.nextLine();
                    manager.removeTask(taskIndex - 1);
                    break;
                case "display":
                    manager.displayTasks();
                    break;
            }
        } while (!command.equals("exit"));
    }
}
```